assets/Product.wxs

```diff
@@ -55,6 +55,31 @@
         shortcut.save
       End If
     </CustomAction>
+
+    <SetProperty Id="RegisterManifest" Before="RegisterManifest" Sequence="execute"
+                 Value ="&quot;[$(var.ProductVersionWithName)]pwsh.exe&quot; -NoProfile -ExecutionPolicy Bypass -File &quot;[$(var.ProductVersionWithName)]RegisterManifest.ps1&quot;" />
```

Comment:
Unnecessary whitespace between `Value` and `=`.  Also, I think how you have the attributes on separate lines (61-66) seems more readable.  You don't need to do it for the entire file (could be separate PR), but for the changes on lines 59-60, it would be great to have that done consistently.

Reply:
Ok. Done. I was thinking about adding the [XML Tools](https://marketplace.visualstudio.com/items?itemName=DotJoshJohnson.xml) extension of VSCode to the recommended VSCode extensions. This extension would allow us to apply consistent XML formatting.